### ISSUE TITLE
nSLOC is not correct

### ISSUE THREAD

**Issue by PatrickAlphaC** [Cyfrin/aderyn#34]
```
git clone https://github.com/Cyfrin/6-thunder-loan-audit
cd 6-thunder-loan-audit
make 
aderyn .
```

You'll see the `nSLOC` output is incorrect

**Comment by alexroan** [comment]
Could you be more specific? What is the expected value, and what output do you get?

**Comment by PatrickAlphaC** [comment]
The output I get is as follows for nSLOC.

```
# Summary

## Files Summary

| Key | Value |
| --- | --- |
| .sol Files | 8 |
| Total nSLOC | 103 |
```

However, this is the correct nSLOC values:

<img width="1477" alt="Screenshot 2023-11-19 at 9 06 57 PM" src="https://github.com/Cyfrin/aderyn/assets/54278053/a10871f7-89be-4e4a-9796-b6e70db434fa">

**Comment by alexroan** [comment]
Could you give access to that repo?

**Comment by alexroan** [comment]
Interesting, the `tokei` package that we use to get details about the source only reads down to line 23 in `src/protocol/ThunderLoan.sol` because of this comment syntax:

```
    /*//////////////////////////////////////////////////////////////
                            STATE VARIABLES
    //////////////////////////////////////////////////////////////*/
```

it's struggling with `/*////////...`

**Comment by alexroan** [comment]
Issue submitted to tokei [here](https://github.com/XAMPPRocky/tokei/issues/1050)

**Comment by alexroan** [comment]
Fixed by #93 ! 🥳
